src/main/app/src/app/shared/progress-snackbar/progress-snackbar.component.html

```diff
@@ -0,0 +1,10 @@
+<!--
+  ~ SPDX-FileCopyrightText: 2021 Atos, 2024 Lifely
```

Comment:
This is a new file right?

Then it should be 
```suggestion
  ~ SPDX-FileCopyrightText: 2024 Lifely
```

:-)

Reply:
This is code from ATOS taken out of util.service.ts; so trivial I guess.

Reply:
This is a new component but the code was ATOS and moved out of util.service.ts